pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -232,6 +232,11 @@ func (c *client) UpdateFunctionFromSource(ctx context.Context, fm FunctionManife
 		return err
 	}
 
+	// Update function configuration.
+	if err := c.updateFunctionConfiguration(ctx, fm); err != nil {
+		return err
+	}
+
```

Comment:
Probably should be at the top of this function (before L230 reading zip) to avoid unnecessary disk read if the update function configuration return error.

Reply:
You're right. Thnaks.